release-notes.txt

```diff
@@ -1,3 +1,401 @@
+    + GitHub feature request template
+    + 'testing' in Github PR template
+    * Rework and documented labeling system
+    * Rework MAINTAINING.md document
+    * Rework CONTRIBUTING.md document
+
+Documentation
+-------------
+    + Add supported radios to OpenThread documentation
+    + New groups under `sys` (Serialization, Math, Memory Management,
+      Hashes)
+    + `arduino` docs for analog map
+    + nanocoap
+    + slstk3401a
+    + slstk3402a
+    + sltb001a
+    + tinycbor
+    * Drivers with `saul` support are now also part of the `drivers_saul` group
+    * Improve `bluepill` and `bmx280` doc
+    * Migrate boards documentation to Doxygen
+
+Core
+----
+    + Architecture dependent thread information
+    * Change `bitarithm`types to `uintXX_t`
+    * Fix `byteorder` bebuftohs() / htobebufs() on big endian
+    * Make `pthread` compilable with g++
+    * Mark OpenODC info variables read-only
+
+System libraries
+----------------
+    + Add EEPROM registration support
+    + Add `arduino` time functions micros() and delayMicroseconds()
+    + ecc: add golay(24,12) half rate and repetition code
+    + Add `fmt` hex byte
+    * Change `base64` api (const + void*)
+    * Configure Arduino serial port at compile time
+    * Move `hamming256` module to generic `ecc`
+    - Remove `cbor` module
+    * sys/auto_init: Fixed initialization of `sht1x`
+
+Networking
+----------
+    + cord_ep: Full CoRE Resource Directory endpoint support
+    + nanocoap: add server-side Block2 support
+    * emb6_sock_udp: copy receive remote correctly
+    * ethos: Add drop frame case to recv function
+    * examples/gcoap: fix post/put without port number
+    * gnrc/6lo: set more data flag on all but last fragment
+    * gnrc_ipv6_nib: always configure 802.15.4 long address with IPv6 included
+    * gnrc_ipv6_nib: fallback to SLAAC if ARO is ignored by upstream
+    * gnrc_ipv6_nib: make automatic NDP packet emission (NS/RS) configurable
+    * gnrc_netif: Fix l2addr_len > 0, but no l2addr edge-case
+    * gnrc_netif: reapply event flags on device reset
+    * gnrc_netif_ieee802154: set FCF Frame Pending when more data is expected
+    * gnrc_rpl: Pass options length to msg validator
+    * gnrc_rpl: fix zeroing of RPL DIS fields
+    * gnrc_sixlowpan_frag: add page context to reassembly buffer
+    * nanocoap: add generic handling for string-based options
+    * nanocoap: allow empty uri/location path option
+
+Packages
+--------
+    + libb2: add support (BLAKE2)
+    + libhydrogen: add support
+    + lora_serialization: LoRa Serialization format support.
+    * tiny-asn1: bump version
+    * openthread: rework of FTD and MTD support
+    * semtech-loramac: bump to version 4.4.1
+    * semtech-loramac: use periph_eeprom to store lorawan configuration
+    * semtech-loramac: add support for RU864 ISM band
+    * semtech-loramac: lorawan support is not considered experimental anymore
+    * semtech-loramac: reduce timeout in timer
```

Comment:
@aabadie I will remove this was a revert of a change introduced by the `semtech_loramac` bump during the release ;)